client/me/concierge/cancel/index.js

```diff
@@ -0,0 +1,73 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+
+/**
+ * Internal dependencies
+ */
+import Main from 'components/main';
+import { localize } from 'i18n-calypso';
+import Confirmation from '../shared/confirmation';
+import Skeleton from './skeleton';
+import { cancelConciergeAppointment } from 'state/concierge/actions';
+import {
+	WPCOM_CONCIERGE_SCHEDULE_ID,
+	CONCIERGE_STATUS_CANCELLED,
+	CONCIERGE_STATUS_CANCELLING_ERROR,
+} from '../constants';
+import { getConciergeSignupForm } from 'state/selectors';
+
+class ConciergeCancel extends Component {
+	static propTypes = {
+		appointmentId: PropTypes.string.isRequired,
+		siteSlug: PropTypes.string.isRequired,
+	};
+
+	getDisplayComponent = () => {
+		let cancellingMessage;
+		const { siteSlug, signupForm, translate } = this.props;
+
+		switch ( signupForm.status ) {
+			case CONCIERGE_STATUS_CANCELLED:
+				cancellingMessage = translate( 'Your Concierge session has been cancelled!' );
+				break;
+
+			case CONCIERGE_STATUS_CANCELLING_ERROR:
```

Comment:
Instead of the error state showing the happy faced guy and a reschedule button, I think it should just show an error message. You can just render a `<Notice status="is-error" text="We couldn't cancel your session, please try again later." />` and nothing else.

https://github.com/Automattic/wp-calypso/blob/master/client/components/notice/index.jsx
https://wpcalypso.wordpress.com/devdocs/design/notice

Reply:
I think an empty page with a notice looks kinda weird. I'll use instead an error card https://wpcalypso.wordpress.com/devdocs/design/cards

https://cloudup.com/cVi06Ygw4Tb

Reply:
Ah yes! The error card is a great idea.